index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <div class="form">
+      <form action="#" id="survey-form">
+        <div>
+          <label for="name" id="name-label">Name: </label>
+          <input type="text" id="name" required placeholder="Enter your name"> 
+        </div>
```

Comment:
You can put every form div into a `<section>` semantic tag. There is no need for the `<div>`